api/src/pcapi/routes/serialization/collective_stock_serialize.py

```diff
@@ -131,7 +131,7 @@ def validate_booking_limit_datetime(  # pylint: disable=no-self-argument
     ) -> Optional[datetime]:
         if (
             booking_limit_datetime
-            and "beginningDatetime" in values
+            and values.get("beginningDatetime", None) is not None
```

Comment:
Le None n'est pas obligatoire (mais `explicit is better than implicit` (et je prefere aussi le mettre)

Reply:
Aligné avec toi, je préfère aussi le mettre !

Je me suis déjà arraché les cheveux pendant 1h parce que j'ai eu pour malheur de choisir 0 en tant qu'identifiant non existant en BDD pour tester une 404 et évidemment un `if payload.offerId` ça ne passait pas 😅 

Là c'est une date mais bon